roles/cinder-common/tasks/ceph_integration.yml

```diff
@@ -17,7 +17,7 @@
 
 - name: fetch rados.py
   get_url: url=https://raw.githubusercontent.com/ceph/ceph/{{ ceph.ceph_version }}/src/pybind/rados.py
-           dest="{{ openstack_package.virtualenv_base }}/cinder/lib/python2.7/site-packages/rados.py"
+           dest="/opt/openstack/current/cinder/lib/python2.7/site-packages/rados.py"
```

Comment:
This feels somewhat wrong hard coding the path.

`dest="{{ lookup('items', 'openstack_package.virtualenv_base') }}/cinder/lib/python2.7/site-packages/rados.py`"

That should work, and matches what we did to fix neutron's rootwrap config.